### ISSUE TITLE
Ignore lobby configureation does not work

### ISSUE THREAD

**Issue by bencebago** [empiricaly/empirica#517]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

Ignore lobby configuration does not work - the game does not load when fewer than the pre-specified number of participants are taking part in the experiment.

### Steps To Reproduce

With any empirica built-in experiment, set the lobby configuration to shared-ignore, with fewer players than PlayerCount, the experiment starts, in the middle of the screen the loading circle appears, but the page never loads. Reproducible with any experiment.

### Empirica Version

```text
Version: v1.9.6
SHA:     12939cc
Build:   193
Branch:  main
Time:    2024-02-11T10:58:50Z

Client:  1.9.6
Server:  1.9.6
```

### What OS are you seeing the problem on?

Windows

### What browser are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by npaton** [comment]
Thanks. I will take a look next end of week. Are you blocked if this is not resolved this week?

**Comment by bencebago** [comment]
I cannot start experimenting until this is resolved.

**Comment by npaton** [comment]
This should be fixed in the latest release (1.9.8). Let me know if you have any issues.
